.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,71 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+
+jobs:
+    php:
+        name: 'Run tests with php ${{ matrix.php-version }}'
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.3'
+                    - php-version: '7.4'
```

Comment:
I remove php 7.2 here because current composer dependencies broken with php7.2. 

```
Run composer validate --strict
./composer.json is valid
No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Error: Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/legacy-factories[v1.0.2, ..., v1.0.3] require illuminate/macroable ^[8](https://github.com/hkulekci/laravel-scout-elasticsearch/runs/5374984938?check_suite_focus=true#step:7:8).0 -> satisfiable by illuminate/macroable[v8.0.0, ..., 8.x-dev], laravel/framework[v8.1[9](https://github.com/hkulekci/laravel-scout-elasticsearch/runs/5374984938?check_suite_focus=true#step:7:9).0, ..., 8.x-dev].
    - laravel/legacy-factories[v1.0.4, ..., v1.0.5] require php ^7.3 -> your php version (7.2.34) does not satisfy that requirement.
    - illuminate/macroable[v8.0.0, ..., v8.[11](https://github.com/hkulekci/laravel-scout-elasticsearch/runs/5374984938?check_suite_focus=true#step:7:11).2] require php ^7.3 -> your php version (7.2.34) does not satisfy that requirement.
    - laravel/legacy-factories[v1.1.0, ..., 1.x-dev] require php ^7.3|^8.0 -> your php version (7.2.34) does not satisfy that requirement.
    - illuminate/macroable[v8.[12](https://github.com/hkulekci/laravel-scout-elasticsearch/runs/5374984938?check_suite_focus=true#step:7:12).0, ..., 8.x-dev] require php ^7.3|^8.0 -> your php version (7.2.34) does not satisfy that requirement.
    - laravel/framework[v8.[19](https://github.com/hkulekci/laravel-scout-elasticsearch/runs/5374984938?check_suite_focus=true#step:7:19).0, ..., 8.x-dev] require php ^7.3|^8.0 -> your php version (7.2.[34](https://github.com/hkulekci/laravel-scout-elasticsearch/runs/5374984938?check_suite_focus=true#step:7:34)) does not satisfy that requirement.
    - Root composer.json requires laravel/legacy-factories ^1.0 -> satisfiable by laravel/legacy-factories[v1.0.2, ..., 1.x-dev].

  Problem 1
    - laravel/legacy-factories[v1.0.2, ..., v1.0.3] require illuminate/macroable ^8.0 -> satisfiable by illuminate/macroable[v8.0.0, ..., 8.x-dev], laravel/framework[v8.19.0, ..., 8.x-dev].
    - laravel/legacy-factories[v1.0.4, ..., v1.0.5] require php ^7.3 -> your php version (7.2.34) does not satisfy that requirement.
    - illuminate/macroable[v8.0.0, ..., v8.11.2] require php ^7.3 -> your php version (7.2.34) does not satisfy that requirement.
    - laravel/legacy-factories[v1.1.0, ..., 1.x-dev] require php ^7.3|^8.0 -> your php version (7.2.34) does not satisfy that requirement.
    - illuminate/macroable[v8.12.0, ..., 8.x-dev] require php ^7.3|^8.0 -> your php version (7.2.34) does not satisfy that requirement.
    - laravel/framework[v8.19.0, ..., 8.x-dev] require php ^7.3|^8.0 -> your php version (7.2.34) does not satisfy that requirement.
    - Root composer.json requires laravel/legacy-factories ^1.0 -> satisfiable by laravel/legacy-factories[v1.0.2, ..., 1.x-dev].

Error: Process completed with exit code 2.
```